### PR TITLE
Update SNMP-Trap-Handler.md

### PR DESCRIPTION
**Pull request by jozefrebjak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

I added a lof of informations how to get working snmptraps.

### CODE REVIEW

File: doc/Extensions/SNMP-Trap-Handler.md [hunk 1]
```diff
@@ -30,23 +50,113 @@ For non-systemd systems, you can edit TRAPDOPTS in the init script in /etc/init.
...
+You can test if your snmptrapd are working correctly very easily, but you need to understand few things.
+
+### Why we need Uptime
+
+When you send a trap, it must of course conform to a set of standards. Every trap needs an uptime value. Uptime is how long the system has been running since boot. Sometimes this is the operating system, other devices might use the SNMP engine uptime. Regardless, a value will be sent.
+
+So what value should you type in the commands below? Oddly enough, simply supplying no value by using two single quotes '' will instruct the command to obtain the value from the operating system you are executing this on.
+
+For those who dig deeper and look at the spooled trap before it's processed will want to understand what type of format it is. Here is an example:
+`DISMAN-EVENT-MIB::sysUpTimeInstance 36:2:40:51.67`
```
Comment (murrant) [inline doc/Extensions/SNMP-Trap-Handler.md]: Worth mentioning that DISMAN-EVENT-MIB::sysUpTimeInstance and SNMPv2-MIB::sysUpTime.0 are the same OID, just depends on the MIBs enabled?

Comment (PipoCanaja) [reply doc/Extensions/SNMP-Trap-Handler.md]: Seems to be the same one indeed. Probably depends on the loaded MIBs. But this sentence is not really mandatory to ease the understanding of traps, so I think removing it is even clearer.

**Comment by jozefrebjak** [comment]
@murrant @PipoCanaja thanks for help with that. Now it’s more cleaner.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-62-release-changelog-march-2020/11423/1

**Pull request merged**